data/json/materials.json

```diff
@@ -1642,6 +1642,116 @@
     "cut_dmg_verb": "scratched",
     "burn_products": [ [ "scrap", 1 ] ]
   },
+  {
+    "type": "material",
+    "id": "lc_steel_chain",
+    "name": "Low Carbon Steel Chain",
+    "//": "SAE 1010/UNS 10100, assumes .1% carbon content and zero slag.",
+    "density": 8,
+    "specific_heat_liquid": 0.82,
+    "specific_heat_solid": 0.45,
+    "latent_heat": 273,
+    "bash_resist": 5,
```

Comment:
Should chain have a lower bash resist than equivalent steel plates? Subjectively, I would think so.

Reply:
Possibly, but doing so compromises the plate armor that subtracts the chain thickness from the plate thickness. I'm not sure how else to reconcile chain and plate in a way that doesn't compromise the armor.